src/ewatercycle/base/model.py

```diff
@@ -452,6 +453,27 @@ def version(self) -> str:
         """Version of the container image."""
         return self.bmi_image.version
 
+    @property
+    def logs(self) -> str:
+        """Return the container logs."""
+        if not hasattr(self, "_bmi"):
+            msg = (
+                "The model has no BMI attached (yet).\n"
+                "Logs will only be available after running `.setup`"
+            )
+            raise ValueError(msg)
+        bmi = self._bmi
+        while isinstance(bmi, OptionalDestBmi | MemoizedBmi):
```

Comment:
This won't work yet for the pcrglobwb plugin due to [this](https://github.com/eWaterCycle/ewatercycle-pcrglobwb/blob/7487c6198e0f03ba8e7a9962510348655142eed5/src/ewatercycle_pcrglobwb/model.py#L23)

the SwapXY class uses https://github.com/eWaterCycle/ewatercycle/blob/main/src/ewatercycle/container.py#L342